### ISSUE TITLE
Failing test: `/tests/plugins/a11yhelp/manual/screenreader`

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#5277]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Run test http://0.0.0.0:1030/tests/plugins/a11yhelp/manual/screenreader

### Expected result

Tester is able to perform step 3.

### Actual result

*What is the actual result of the above steps?*

It is impossible to use `ALT+0` on mobile device. Also, we don't support screen readers on mobile devices.

## Other details

* Browser: Chrome mobile 103 (Android)
* OS: Android 11
* CKEditor version: 4.19.1 (pre-release)
* Installed CKEditor plugins: full all

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5300
